pyslackersweb/website/views.py

```diff
@@ -7,6 +8,9 @@
 from .models import InviteSchema
 from .tasks import GITHUB_REPO_CACHE_KEY, SLACK_COUNT_CACHE_KEY, SLACK_TZ_CACHE_KEY
 
+
+LOG = logging.getLogger(__name__)
```

Comment:
Can you wrap the logger in the `ContextAwareLoggingAdapter` (like [this](https://github.com/pyslackers/website/blob/master/pyslackersweb/website/tasks.py#L16))? 

That adds a request context UUID so we can trace the logs more easily.

Also this is a weird knit pick of mine, but the other spots the logger is used it’s called `logger`, I’d prefer to be consistent

Reply:
Done,
I wonder if the `ContextAwareLoggingAdapter` works fine with sentry. We might want to add the uuid to the `extra` instead. To be tested I guess

Reply:
It may be less necessary with sentry, though it’s nice for larger apps when tracing logs that don’t result in errors.